cli/ceph/orch/tuned_profile.py

```diff
@@ -0,0 +1,61 @@
+from cli import Cli
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class TunedProfile(Cli):
+    def __init__(self, nodes, base_cmd):
+        super(TunedProfile, self).__init__(nodes)
+        self.base_cmd = f"{base_cmd} tuned-profile"
+
+    def apply(self, spec_file, **kw):
+        """
+        apply tuned profile.
+        Args:
+            spec_file: tune spec file details
+            kw (dict): Key/value pairs that needs to be provided to the installer.
+        """
+        cmd = f"{self.base_cmd} apply -i {spec_file}"
+        out = self.execute(sudo=True, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def list(self):
+        """
+        list tuned profile.
+        """
+        cmd = f"{self.base_cmd} ls"
+        out = self.execute(sudo=True, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].split("\n")
+        return out[0]
+
+    def remove(self, **kw):
+        """
+        remove tuned profile.
+        Args:
+            kw (dict): Key/value pairs that needs to be provided to the installer.
+        """
+        profile_name = kw["profile_name"]
+        cmd = f"{self.base_cmd} rm {profile_name}"
+        out = self.execute(sudo=True, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def modify(self, **kw):
+        """
+        modify tuned profile.
+        Args:
+            kw (dict): Key/value pairs that needs to be provided to the installer.
```

Comment:
Same as above, add mandatory parameter to method signature

Reply:
Updated